lib/stub_ui/lib/src/ui/text.dart

```diff
@@ -1709,6 +1777,58 @@ class ParagraphBuilder {
     _ops.add(text);
   }
 
+  /// Adds an inline placeholder space to the paragraph.
+  ///
+  /// The paragraph will contain a rectangular space with no text of the dimensions
+  /// specified.
+  ///
+  /// The `width` and `height` parameters specify the size of the placeholder rectangle.
+  ///
+  /// The [alignment] parameter specifies how the placeholder rectangle will be vertically
```

Comment:
surround alignment in ` instead of [], since it is just a parameter and not a property?

Reply:
Same for baseline, baslineOffset (2x) and height below?